### PR TITLE
Add option for Ionosphere correction to the INSAR_ISCE_TESTS job spec

### PR DESCRIPTION
**Pull request by cmarshak**
@asjohnston-asf and @jhkennedy - this is a sample of what I am hoping for. I merged the new CLI into dev, but there is significantly more testing that has to be done.

I hope I can learn a few things as you update hyp3.

### CODE REVIEW

**Comment by cmarshak** [comment]
The reference to the new parameters is here: https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/blob/dev/isce2_topsapp/__main__.py#L94-L97

File: job_spec/INSAR_ISCE_TEST.yml [hunk 1]
```diff
@@ -27,6 +27,27 @@ INSAR_ISCE_TEST:
...
+        description: The bounding box of the region of interest as xmin, ymin, xmax, ymax
+        example: [-121.167298, 33.929114, -118.055825, 35.904876]
+        items:
+          description: the x or y coordinates of the bounding box
+          type: float
+    frame_id:
+      api_schema:
+        default: -1
+        type: int
+        description: the frame-id derived from ARIA-S1-GUNW fixed frame convention; if specified, region of interest, must be too.
```
Comment (jhkennedy) [inline job_spec/INSAR_ISCE_TEST.yml:50]:
```suggestion
```

File: job_spec/INSAR_ISCE_TEST.yml [hunk 2]
```diff
@@ -53,6 +74,12 @@ INSAR_ISCE_TEST:
         - Ref::granules
         - --secondary-scenes
         - Ref::secondary_granules
+        - --region-of-interest
+        - Ref::region_of_interest
+        - --estimate-ionosphere-delay
+        - Ref::estimate_ionosphere_delay
+        - --frame-id
+        - Ref::frame_id
```
Comment (jhkennedy) [inline job_spec/INSAR_ISCE_TEST.yml:82]:
```suggestion
        - --estimate-ionosphere-delay
        - Ref::estimate_ionosphere_delay
```

**Review by jhkennedy — CHANGES_REQUESTED** [review]
Let's separate out the framing from the ionosphere. 

we can address the ionosphere here (issue #1451) here while framing (issue #1452) will be addressed in #1464

**Review by jtherrmann — APPROVED** [review]

**Pull request merged**